src/cascade/saver/db/save_model_results_to_db.py

```diff
@@ -90,9 +90,12 @@ def _normalize_draws_df(draws_df, execution_context):
     node_to_location = {r.node_id: r.c_location_id for _, r in node_table.iterrows()}
 
     draws["location_id"] = draws.node_id.apply(lambda nid: node_to_location[nid])
-
-    draws["sex_id"] = draws.x_sex.apply(lambda x: {-0.5: 2, 0.5: 1}[x])
-    return draws.drop(["node_id", "weight_id", "x_sex"], "columns")
+    covariate_table = execution_context.dismodfile.covariate
+    sex_index = int(covariate_table[covariate_table.covariate_name == "sex"].covariate_id)
```

Comment:
What happens if there are multiple sex covariates? Or are we assuming that is sufficiently impossible?

Reply:
Every covariate is saved only once because of the machinery I made. So if you have sex used by two covariate multipliers, then they use the same column. If sex is transformed, it becomes sex_log or sex_squared, which then gets its own column.